code/modules/clothing/suits/labcoat.dm

```diff
@@ -22,12 +22,19 @@
 	icon_state = "labcoat_emt"
 	item_state = "labcoat_cmo"
 
-/obj/item/clothing/suit/toggle/labcoat/brig_phys
+//Brig Phys
+/obj/item/clothing/suit/toggle/labcoat/brigphys
 	name = "security medic's labcoat"
 	icon_state = "labcoat_sec"
 	item_state = "labcoat_sec"
 	armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 10, rad = 0, fire = 50, acid = 50)
 
+/obj/item/clothing/suit/toggle/labcoat/brigphys/vest
+	name = "brig physician's vest"
+	desc = "A lightweight vest worn by the Brig Physician."
+	icon_state = "brig_phys_vest"
```

Comment:
This icon state doesn't support toggleability

Reply:
oof, I'll revert that

Reply:
PR should be ready now